code/modules/clothing/spacesuits/rig/suits/station.dm

```diff
@@ -232,11 +241,6 @@ Advanced Voidsuit: Technomancer Exultant
 
 	helm_type = /obj/item/clothing/head/helmet/space/rig/hazard
 
-	allowed = list(
-		/obj/item/weapon/gun,/obj/item/device/lighting/toggleable/flashlight,/obj/item/weapon/tank,
-		/obj/item/device/suit_cooling_unit,/obj/item/weapon/melee/baton
-	)
-
```

Comment:
Guns and baton now diallowed for hazard rigs?

Reply:
no they're in the base list